blog/update_v1.24/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: layouts/news/single.njk
+title: "Pentacode v1.24: Noch mehr Power im Bereich Lohnvorbereitung und Datenexport"
+date: 2023-02-07
+cover: datenexport.webp
+author: jseidel
+categories:
+  - Produkt
+---
+
+Auch wenn sich der Kampf mit der Buchhaltung manchmal wie eine Sysyphusarbeit anfühlt - die Aufbereitung und Dokumentation von Lohnberichten und co. ist ein integraler Teil des Unternehmertums, der (richtig unternommen) viele Früchte abwerfen kann. 
+Mit Pentacode V. 1.24 helfen wir Ihnen den Bürokratiedschungel besser zu navigieren und wertvolle Zeit bei der Vorbereitung von Lohndaten zu sparen. Dabei werden Ihnen **eine gänzlich überarbeitete Datenexportfunktion, viele neue Individualisierungsoptionen sowie viele weitere Verbesserungen & Neuheiten** helfen die Früchte einer guten Buchhaltung einfacher zu ernten. 
+
+## Datenexport
+
+Der neue Datenexportreiter - das Herzstück unseres neuen Updates. Statt wie zuvor Stammdatenblatt, Lohnberichte, Arbeitszeiten und co. verteilt über die verschiedenen Funktionen Pentacodes suchen zu müssen, finden Sie nun alle benötigten Daten gut strukturiert an einem Ort - Sie müssen nur noch auf Download klicken.
+
+{%figure caption="Die Lohndaten für eine ganze Abteilung für die Woche bereitstellen? Mit dem neuen Buchhaltungsfeature aus Update 1.24 kein Problem." %}
+<img src="lohnberichte.gif"/>
```

Comment:
Bitte zu den ganzen Bildern noch Alt-Texte hinzufügen. Dabei musst du dir allerdings nicht zu viel Mühe geben, geht hauptsächlich darum, überhaupt einen Alt-Text zu haben wegen SEO. Also einfach eine kurze Beschreibung, was in dem Bild zu sehen ist. Z.B.:

```suggestion
<img src="lohnberichte.gif" alt="Animierter Screenshot der Datenexport-funktion." />
```